.github/workflows/release-please.yml

```diff
@@ -0,0 +1,47 @@
+# Automate releases of new app versions
+name: Release new app version
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
+          package-name: graasp-app-text-input
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"docs","section":"Documentation","hidden":false},{"type":"test","section":"Tests","hidden":false}]'
```

Comment:
Would it make sense to factor out release please to our own actions registry at some point? So that we can set these default changelog-types, and add the possibility to overwrite the config?

You don't need to fix this now :sweat_smile:

Reply:
It is an interesting idea, we should discuss it in more detail !